validate/validate.go

```diff
@@ -424,7 +424,11 @@ func (v *Validator) CheckPlatform() (msgs []string) {
 		}
 	}
 
-	if v.platform != "windows" {
+	if v.platform == "windows" {
+		if v.spec.Windows == nil {
+			msgs = append(msgs, "'windows' MUST be set when platform is `windows`")
+		}
+	} else {
 		if v.spec.Windows != nil {
 			msgs = append(msgs, fmt.Sprintf("'windows' MUST NOT be set when target platform is %q", v.platform))
```

Comment:
> This is no longer true in rc6.

And that is getting fixed by #416.  Once #416 lands, the only remaining issue will be that we're using v6 logic but [requiring][1] [1.0.0rc5][2], and that's not a big deal in between runtime-tools releases.

[1]: https://github.com/opencontainers/runtime-tools/blob/0d35fc8437ec8292cd7065e332000c584f7024c5/validate/validate.go#L164-L165
[2]: https://github.com/opencontainers/runtime-tools/blob/0d35fc8437ec8292cd7065e332000c584f7024c5/vendor/github.com/opencontainers/runtime-spec/specs-go/version.go#L6-L18